CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.13)
+
+option(FIZZY_TESTING "Enable Fizzy internal tests")
+
+project(fizzy LANGUAGES CXX)
+
+set(include_dir ${PROJECT_SOURCE_DIR}/include)  # Public include directory.
```

Comment:
Not used any more, but should be included in the fizzy lib when we will have any public header. This can be removed from this PR.